pandas/tests/reshape/merge/test_merge.py

```diff
@@ -1826,3 +1834,51 @@ def test_merge_on_indexes(self, left_df, right_df, how, sort, expected):
                           how=how,
                           sort=sort)
         tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize('index',
+                         [
```

Comment:
write like

```
@pytest.mark.parametrize(
    'index',
     [
```

so it is closer to the left margin, then each of the indexes can be writen more simply, also don't use the data kwarg
e.g.
```Index(['A', 'B'], name='index_col')``